invenio/utils/persistentid.py

```diff
@@ -297,6 +303,18 @@ def is_pmcid(val):
     return pmcid_regexp.match(val)
 
 
+def is_gnd(val):
+    """Test if argument is a GND Identifier."""
+    if val.startswith('http://d-nb.info/gnd/'):
```

Comment:
Can you assign this string to a module level variable (similar to the regexs) and use that variable instead (i.e. to ensure any change will be propagated correctly).